### PR TITLE
Preserve UTF-8 storage update arguments

### PR DESCRIPTION
**Pull request by Jim8y**
## Summary
- Convert plain `contract storage update` text arguments to UTF-8 before persisting them.
- Keep existing base64 storage argument handling unchanged.
- Add focused workflow validation tests for Unicode text and base64 inputs.

## Validation
- `dotnet test test/test.workflowvalidation/test.workflowvalidation.csproj --configuration Release --no-restore --verbosity minimal --filter ContractStorageUpdateCommandTests`
- `dotnet test neo-express.sln --configuration Release --no-restore --verbosity minimal --filter "FullyQualifiedName!~test.workflowvalidation.WorkflowValidationTests&FullyQualifiedName!~test.workflowvalidation.NeoxpToolIntegrationTests&FullyQualifiedName!~test.workflowvalidation.NeoxpAdvancedIntegrationTests"`
- `git diff --check`

### CODE REVIEW

File: test/test.workflowvalidation/ContractStorageUpdateCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+using NeoExpress.Commands;
+using System.Text;
+using Xunit;
+
+namespace test.workflowvalidation;
+
+public class ContractStorageUpdateCommandTests
+{
+    [Fact]
+    public void ConvertStorageArgument_preserves_unicode_text_as_utf8()
```
Comment (Wi1l-B0t) [inline test/test.workflowvalidation/ContractStorageUpdateCommandTests.cs]:
```suggestion
    public void ConvertStorageArgument_PreservesUnicodeTextAsUtf8()
```

File: test/test.workflowvalidation/ContractStorageUpdateCommandTests.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        var parser = new ContractParameterParser(ProtocolSettings.Default);
+        const string value = "neo-汉字";
+
+        var result = ContractCommand.Storage.StorageUpdateKeyValue.ConvertStorageArgument(parser, value);
+
+        result.Should().Be(Convert.ToBase64String(Encoding.UTF8.GetBytes(value)));
+    }
+
+    [Fact]
+    public void ConvertStorageArgument_preserves_base64_input()
```
Comment (Wi1l-B0t) [inline test/test.workflowvalidation/ContractStorageUpdateCommandTests.cs]:
```suggestion
    public void ConvertStorageArgument_PreservesBase64Input()
```

**Review by shargon — APPROVED** [review]

**Review by ajara87 — APPROVED** [review]
